Add Python 3.8 support

milljm opened the pull request:
Remove need to download a file (I didn't know what I was doing back then).
Properly test imports of all python packages involved.

TODO: Once this is in, we should clean up Idaholab channel. I think all the
moose-tools available may be causing slow `conda install` solves.

Closes #16620

Commit:
Add Python 3.8 support

Remove need to download a file (I didn't know what I was doing back then).
Properly test imports of all python packages involved.

TODO: Once this is in, we should clean up Idaholab channel. I think all the
moose-tools available may be causing slow `conda install` solves.

Closes #16620

Commit:
Simplify moose-tools recipe

Simply the recipe. Also, attempt changes to possibly help with conda install solve time.

Refs: #16620

Commit:
Add noarch: python

additional attempts to streamline the recipe (and decrease solve times)

Commit:
Only build on one machine

Add osx to the list of machines to skip building this package on. Now that it is a
noarch, it only needs to be built once, on any machine. I nominate meowbuild (linux).

Commit:
Simplify the skip routine

Do not skip when building on linux machines.

Commit:
Simple change to trigger a build

Need to clear out the now faulty hpcsc channel with a bad moose-tools package. Not sure
how we can automate this with each new commit. ...will think on this...

moosebuild commented:
Job [Documentation](https://civet.inl.gov/job/654625/) on 660068d wanted to post the following:

View the site [here](https://mooseframework.inl.gov/docs/PRs/16621/site)

This comment will be updated on new commits.

Commit:
Not ready to support 3.8

While this ticket was supposed to be about supporting 3.8, now it's about conda's slow solve time.

milljm commented:
Closing. This task is being superseded by #17106
